src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -411,6 +411,25 @@ contract AllocationManagerUnitTests_SlashOperator is AllocationManagerUnitTests
         allocationManager.slashOperator(_randSlashingParams(random().Address(), 0));
     }
 
+    function test_revert_CannotSlashBeaconChainETHStrategy() public {
+        IStrategy[] memory strategies = allocationManager.beaconChainETHStrategy().toArray();
+
+        cheats.prank(defaultAVS);
+        allocationManager.addStrategiesToOperatorSet(defaultOperatorSet.id, strategies);
+
+        cheats.prank(defaultAVS);
+        allocationManager.slashOperator(SlashingParams(defaultOperator, defaultOperatorSet.id, WAD, ""));
+
+        // assertEq(
```

Comment:
Can we slash this operator (mag < WAD) and then assert that the magnitude is still WAD